src/script/lua_api/l_object.cpp

```diff
@@ -1279,8 +1283,9 @@ int ObjectRef::l_get_fov(lua_State *L)
 	PlayerFovSpec fov_spec = player->getFov();
 	lua_pushnumber(L, fov_spec.fov);
 	lua_pushboolean(L, fov_spec.is_multiplier);
+	lua_pushnumber(L, fov_spec.transition_time);
```

Comment:
Hmm, is this even of any use? Mods don't know if/how much of the transition time has passed.

Reply:
This value doesn't communicate *when* the actual transition happened, but it at least serves to allow mods to know what the player's last transition time was. But at the moment, it's mostly useless.

Reply:
I added this, in case there could be a use for this in the future. Perhaps we can add this back when this is actually needed, and remove it for now?

Reply:
Dunno, can't really hurt to return it.